koku/cost_models/serializers.py

```diff
@@ -394,6 +395,12 @@ class Meta:
         choices=metric_constants.DISTRIBUTION_CHOICES, required=False, allow_blank=True
     )
 
+    currency = serializers.ChoiceField(
+        choices=tuple([(currency.get("code"), currency.get("code")) for currency in CURRENCIES]),
+        required=False,
+        allow_blank=True,
+    )
```

Comment:
Is this a list of currencies?

In the UI, we just need the API to return a single currency -- the currency used to create the cost model.